columnflow/calibration/cms/egamma.py

```diff
@@ -29,6 +29,8 @@
 @dataclass
 class EGammaCorrectionConfig:
     correction_set: str = "Scale"
+    compound: bool = False
+    corrector_strings: dict[str] = field(default_factory=dict)
     corrector_kwargs: dict[str, Any] = field(default_factory=dict)
```

Comment:
I think the existing `corrector_kwargs` can already handle what `corrector_strings` are supposed to do. Also, it would be good to have a default here.

```suggestion
    compound: bool = False
    corrector_kwargs: dict[str, Any] = field(default_factory=lambda: {"type": ...})
```

Reply:
At first glance this would be true, since we use `corrector_kwargs` as dynamic extension of `variable_map`, and the correction and its uncertainty share the same keyword within the corrector I splitted this up. 

But right when I wrote this reply I thought of a way to apply this.

Reply:
As discussed offline - we added concrete arguments instead of using corrector_kwargs. This is more explicit and forces the user to hand these values.